resources/js/components/Common/TimeLeft.vue

```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts">
+
+import {ref} from 'vue'
+import {calcSecondsLeftToDate, isTimeout, secondsToHour, timeToString} from '@/Helpers/Time'
+
+const text = ref('')
+const emit = defineEmits(['timeout'])
+const props = defineProps<{ to?: string | number }>()
+
+function updateTimer() {
```

Comment:
Maybe better idea will be create some computed property for preparing display time in real time ?
https://vuejs.org/guide/essentials/computed.html#computed-properties

Reply:
done